app/cdap/components/GridTable/components/NoRecordScreen/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Please make sure to add Copyright header content at the beginning for every file.